src/WebSocketManager/WebSocketManagerMiddleware.cs

```diff
@@ -23,7 +23,10 @@ public class WebSocketManagerMiddleware
         public async Task Invoke(HttpContext context)
         {
             if (!context.WebSockets.IsWebSocketRequest)
+            {
+                await _next.Invoke(context);
```

Comment:
Aaah, totally missed that, sorry!

I will review this once I (hope to) get some time on my hands!
Thanks a lot for your contribution!